viz.yaml

```diff
@@ -64,17 +86,23 @@ fetch:
     refetch: TRUE
     metadata: "Updates every 6 hours on http://www.nhc.noaa.gov/gis/"
     comment: disabling refetch until timestamp is hooked up
-  -
-    id: precip-data
-    fetcher: precip
-    start.date: "2017-09-07"
-    end.date: "2017-09-15"
-    states: ["Florida", "Georgia", "South Carolina"]
-    scripts: scripts/fetch/fetchPrecip.R
-    location: cache/precip.csv
-    mimetype: text/csv
-    refetch: true
-    comment: disabling refetch to avoid too many GDP requests
 process:
 visualize:
+  -
+    id: base-map
+    location: cache/base-map.svg
+    visualizer: svg_base_map
+    depends: ["view-limits", "states", "counties"]#, "storm-track", 'storm-location', "gage-sites"]
+    scripts: scripts/visualize/visualize_utils.R
+    mimetype: image/svg+xml
+  -
+    id: hurricane-map
+    location: figures/hurricane-map.svg
+    visualizer: hurricane_map
+    depends: ["base-map"]#, "precip-category", "precip-colors", "discharge-sparks"]
```

Comment:
we want to be using named depends here, if there is going to be multiple

Reply:
see https://github.com/USGS-VIZLAB/internal-analytics/blob/master/viz.yaml#L210

Reply:
I think I need a primer on how to do that correctly.